packages/node_modules/@webex/plugin-meetings/src/personal-meeting-room/index.js

```diff
@@ -0,0 +1,62 @@
+import {StatelessSparkPlugin} from '@ciscospark/spark-core';
+
+import {MEETINGS, PERSONAL_ROOM} from '../constants';
+import MeetingInfoRequest from '../meeting-info/request';
+
+/**
+ * @class PersonalMeetingRoom
+ */
+export default class PersonalMeetingRoom extends StatelessSparkPlugin {
+  namespace = MEETINGS;
+
+  /**
+   *
+   * @param {Object} attrs
+   * @param {Object} options
+   */
+  constructor(attrs, options) {
+    super({}, options);
+    this.pmr = undefined;
+    this.sipUri = undefined;
+    this.link = undefined;
+    this.userId = undefined;
+    this.name = undefined;
```

Comment:
We discuss live initializing with `undefined` instead of `null`. Feels like you are defining something with `undefined` as a value.

Reply:
yeah i didnt see yours updated everything to null, so was trying to leave it consistent,
I agree with you as per stuff like this https://www.ajaymatharu.com/javascript-difference-between-undefined-and-null/